README
NAME
    Task::BeLike::RSRCHBOY::Web - Web-related modules RSRCHBOY uses!

VERSION
    version 0.001

DESCRIPTION
    This is a task package cataloging the web-related modules I often use.
    I've broken it out of Task::BeLike::RSRCHBOY so as to reduce that
    package's footprint on machines where these are not needed.

TASK CONTENTS
  Other Tasks
   Task::Catalyst
   Task::BeLike::RSRCHBOY 0.002
  Catalyst
   Catalyst::Runtime 5.9
   Catalyst::Devel
   CatalystX::SimpleLogin
   CatalystX::RoleApplicator
   Catalyst::Plugin::Authentication
   Catalyst::Plugin::Authorization::ACL
   Catalyst::Plugin::Authorization::Roles
   Catalyst::Plugin::AutoCRUD 1.112560
   Catalyst::Plugin::Session
   Catalyst::Plugin::Session::State::Cookie
   Catalyst::Plugin::Session::Store::File
   Catalyst::TraitFor::Request::BrowserDetect
   Catalyst::TraitFor::Request::REST::ForBrowsers
   Catalyst::Controller::REST
   Catalyst::Model::DBIC::Schema 0.59
   Catalyst::View::TT
  Dancer
   Dancer
  Plack / PSGI
   Plack
   Plack::Middleware::Debug
   Plack::Middleware::SetAccept
   Starlet
   Starman
  Forms
   HTML::FormHandler
  Templating / etc
   HTML::Builder 0.006
   Text::Haml
   Template::Toolkit
   Template::Plugin::JSON::Escape
AUTHOR
    Chris Weyl <cweyl@alumni.drew.edu>

COPYRIGHT AND LICENSE
    This software is Copyright (c) 2012 by Chris Weyl.

    This is free software, licensed under:

      The GNU Lesser General Public License, Version 2.1, February 1999



Changes
Revision history for Task-BeLike-RSRCHBOY-Web

0.001     2012-03-30 21:20:16 America/Los_Angeles


lib/Task/BeLike/RSRCHBOY/Web.pm
#
# This file is part of Task-BeLike-RSRCHBOY-Web
#
# This software is Copyright (c) 2012 by Chris Weyl.
#
# This is free software, licensed under:
#
#   The GNU Lesser General Public License, Version 2.1, February 1999
#
package Task::BeLike::RSRCHBOY::Web;
{
  $Task::BeLike::RSRCHBOY::Web::VERSION = '0.001';
}

# ABSTRACT: Web-related modules RSRCHBOY uses!

!!42;



=pod

=head1 NAME

Task::BeLike::RSRCHBOY::Web - Web-related modules RSRCHBOY uses!

=head1 VERSION

version 0.001

=head1 DESCRIPTION

This is a task package cataloging the web-related modules I often use.  I've
broken it out of L<Task::BeLike::RSRCHBOY> so as to reduce that package's
footprint on machines where these are not needed.

=head1 TASK CONTENTS

=head2 Other Tasks

=head3 L<Task::Catalyst>

=head3 L<Task::BeLike::RSRCHBOY> 0.002

=head2 Catalyst

=head3 L<Catalyst::Runtime> 5.9

=head3 L<Catalyst::Devel>

=head3 L<CatalystX::SimpleLogin>

=head3 L<CatalystX::RoleApplicator>

=head3 L<Catalyst::Plugin::Authentication>

=head3 L<Catalyst::Plugin::Authorization::ACL>

=head3 L<Catalyst::Plugin::Authorization::Roles>

=head3 L<Catalyst::Plugin::AutoCRUD> 1.112560

=head3 L<Catalyst::Plugin::Session>

=head3 L<Catalyst::Plugin::Session::State::Cookie>

=head3 L<Catalyst::Plugin::Session::Store::File>

=head3 L<Catalyst::TraitFor::Request::BrowserDetect>

=head3 L<Catalyst::TraitFor::Request::REST::ForBrowsers>

=head3 L<Catalyst::Controller::REST>

=head3 L<Catalyst::Model::DBIC::Schema> 0.59

=head3 L<Catalyst::View::TT>

=head2 Dancer

=head3 L<Dancer>

=head2 Plack / PSGI

=head3 L<Plack>

=head3 L<Plack::Middleware::Debug>

=head3 L<Plack::Middleware::SetAccept>

=head3 L<Starlet>

=head3 L<Starman>

=head2 Forms

=head3 L<HTML::FormHandler>

=head2 Templating / etc

=head3 L<HTML::Builder> 0.006

=head3 L<Text::Haml>

=head3 L<Template::Toolkit>

=head3 L<Template::Plugin::JSON::Escape>

=head1 AUTHOR

Chris Weyl <cweyl@alumni.drew.edu>

=head1 COPYRIGHT AND LICENSE

This software is Copyright (c) 2012 by Chris Weyl.

This is free software, licensed under:

  The GNU Lesser General Public License, Version 2.1, February 1999

=cut


__END__



Makefile.PL

use strict;
use warnings;

use 5.006;

use ExtUtils::MakeMaker 6.30;



my %WriteMakefileArgs = (
  "ABSTRACT" => "Web-related modules RSRCHBOY uses!",
  "AUTHOR" => "Chris Weyl <cweyl\@alumni.drew.edu>",
  "BUILD_REQUIRES" => {
    "File::Find" => 0,
    "File::Temp" => 0,
    "Test::More" => "0.88",
    "Test::Requires" => 0,
    "strict" => 0,
    "warnings" => 0
  },
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => "6.30"
  },
  "DISTNAME" => "Task-BeLike-RSRCHBOY-Web",
  "EXE_FILES" => [],
  "LICENSE" => "lgpl",
  "NAME" => "Task::BeLike::RSRCHBOY::Web",
  "PREREQ_PM" => {
    "Catalyst::Controller::REST" => 0,
    "Catalyst::Devel" => 0,
    "Catalyst::Model::DBIC::Schema" => "0.59",
    "Catalyst::Plugin::Authentication" => 0,
    "Catalyst::Plugin::Authorization::ACL" => 0,
    "Catalyst::Plugin::Authorization::Roles" => 0,
    "Catalyst::Plugin::AutoCRUD" => "1.112560",
    "Catalyst::Plugin::Session" => 0,
    "Catalyst::Plugin::Session::State::Cookie" => 0,
    "Catalyst::Plugin::Session::Store::File" => 0,
    "Catalyst::Runtime" => "5.9",
    "Catalyst::TraitFor::Request::BrowserDetect" => 0,
    "Catalyst::TraitFor::Request::REST::ForBrowsers" => 0,
    "Catalyst::View::TT" => 0,
    "CatalystX::RoleApplicator" => 0,
    "CatalystX::SimpleLogin" => 0,
    "Dancer" => 0,
    "HTML::Builder" => "0.006",
    "HTML::FormHandler" => 0,
    "Plack" => 0,
    "Plack::Middleware::Debug" => 0,
    "Plack::Middleware::SetAccept" => 0,
    "Starlet" => 0,
    "Starman" => 0,
    "Task::BeLike::RSRCHBOY" => "0.002",
    "Task::Catalyst" => 0,
    "Template::Plugin::JSON::Escape" => 0,
    "Template::Toolkit" => 0,
    "Text::Haml" => 0
  },
  "VERSION" => "0.001",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



